src/Logic.hs

```diff
@@ -1,13 +1,95 @@
 module Logic where
 
 import Data.Array
+import Foreign.Marshal.Unsafe
 import Game
+import Graphics.Gloss.Interface.Pure.Game
```

Comment:
Para que y en donde estas utilizando este import?

Reply:
That is for reading the input keyboard.